src/components/LoginForm.js

```diff
@@ -4,19 +4,49 @@ import SignupButton from "../components/SignupButton";
 import TextFieldContainer from "../components/TextFieldContainer";
 import styled from "styled-components";
 
-const LoginFormStyled = styled.div`
-  display: flex;
+const LoginFormStyled = styled.form`
   width: 100vw;
   height: 100vh;
 `;
 
+const DivEmail = styled.div`
```

Comment:
The naming convention is a bit unusual and inconsistent sometimes `...Container` and sometimes `Div...`. I would prefer the first option.